connectivity/drivers/cellular/TELIT/ME310/TELIT_ME310_CellularStack.cpp

```diff
@@ -25,6 +25,10 @@ static const int sslctxID = 1;
 
 using namespace mbed;
 
+namespace mbed {
+constexpr auto ME310_SOCKET_TIMEOUT = std::chrono::milliseconds(1000);
```

Comment:
```suggestion
constexpr auto ME310_SOCKET_TIMEOUT = 1s;
```
I think this also works.
Maybe leave it in the header file as macro? I think all caps is used for macros.